NitroxModel/DataStructures/GameLogic/NitroxQuaternion.cs

```diff
@@ -146,15 +146,16 @@ private static NitroxVector3 NormalizeAngles(NitroxVector3 angles)
 
         private static float NormalizeAngle(float angle)
         {
-            while (angle >= 360)
+            while (angle < 0)
             {
-                angle -= 360;
+                angle += 360;
             }
 
-            while (angle < 0)
+            while (angle >= 360)
             {
-                angle += 360;
+                angle -= 360;
             }
+
             return angle;
         }
```

Comment:
Why not use modulus operator here? `return angle % 360;`

Reply:
does modulus work like that?